keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk

```diff
@@ -0,0 +1,6 @@
+# MCU name
+MCU = RP2040
+BOOTLOADER = rp2040
+BOARD = GENERIC_RP_RP2040
+
```

Comment:
Can also go in info.json like pi60_hse
```suggestion
```

Reply:
I was having difficulty getting that to work properly when I have options for multiple types of controllers